### PR TITLE
Update PS4 Docs

### PR DESCRIPTION
**Pull request by ktnrg45**
**Description:**
Added additional config info.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/ps4.markdown [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: page
 title: "Sony PlayStation 4"
 description: "Instructions on how to integrate a Sony PlayStation 4 into Home Assistant."
-date: 2019-02-12 01:08
+date: 2019-03-15 01:08
```
Comment (klaasnicolaas) [inline source/_components/ps4.markdown]: Please do not change this

File: source/_components/ps4.markdown [hunk 2]
```diff
@@ -53,9 +52,40 @@ There are varying methods to perform this, dependent on your OS that is running
...
+
+- Debian
+- Hassbian
+- Rassbian
+- Armbian
+- Ubuntu
+
+In terminal run the following command:
+
+`$ sudo setcap 'cap_net_bind_service=+ep' <python>`
```
Comment (klaasnicolaas) [inline source/_components/ps4.markdown]: Please use [this kind](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) of format for code in markdown.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/ps4.markdown (hunk 1 above)
Comment (ktnrg45) [reply source/_components/ps4.markdown]: Ok. Thanks. Reverted.

File: source/_components/ps4.markdown (hunk 2 above)
Comment (ktnrg45) [reply source/_components/ps4.markdown]: Removed $ prefix from code

Comment (klaasnicolaas) [reply source/_components/ps4.markdown]: I think you must do more then that, I will change somthing as example

Comment (ktnrg45) [reply source/_components/ps4.markdown]: How about now?

**Pull request closed without merging**